src/runtime_src/core/tools/common/tests/TestPreeemptionOverhead.cpp

```diff
@@ -177,5 +179,7 @@ TestPreemptionOverhead::run(std::shared_ptr<xrt_core::device> dev)
       ptree.put("status", XBU::test_token_passed);
     }
   }
+  // Restore the original preemption state
+  xrt_core::device_update<xq::preemption>(dev.get(), static_cast<uint32_t>(layer_boundary));
```

Comment:
On Linux, enable layer_boundary preemption requires root privilege. Could you add a try-catch and if it fails due to EPERM, issue a warning and skip the test?

Reply:
I see. So, the rule is that you need to use --verbose in order to see skipped cases? I'm Ok with that.